### DISCUSSION TITLE
Keyball 44の特定のキー反応不良について

### DISCUSSION THREAD

**Discussion by don-wang** [Yowkees/keyball#501] [Q&A] [answered]
お世話になっております。
Keyball 44を組立てて、大変楽しんでおります。
しかし、最近から右手のYキー（添付画像赤い丸の部分）が反応不良になりました。
スイッチ差し直したら、一時的に普通に動作しますが、少し時間立つと反応しなくなります。
ダイオードとスイッチも交換したが改善できませんでした。
お忙しいところ恐縮ですがアドバイスであればご教示ください。
よろしくお願いいたします。
![keyball-r](https://github.com/Yowkees/keyball/assets/1377797/fdb1e1ed-cb38-4a1a-8658-c314779ba426)

**Comment by Ryo624** [comment, marked as answer]
ご連絡ありがとうございます。

Yキーのみの反応の不具合ということで、キーソケットもしくはダイオードが原因となっている可能性が高いので、キーソケットとパッドの間にはんだがしっかり馴染むように根気よくはんだ付けをやり直すことと、キーに対応するダイオードの足が浮かないようパッドに接続るようにはんだ付けをやり直してみてください。
キースイッチの足折れがないかもご確認ください。

**Comment by don-wang** [comment]
返事が遅くなり申し訳ありません。
ご指摘したとおりにキーソケットとダイオードのはんだをし直したら改善できました。
該当のダイオードがどれかがわからなくて、一応キーソケット下の二つ全部やりました。
ありがとうございました。

**Comment by Ryo624** [comment]
対応するダイオードの位置をご指摘しておらず申し訳ありませんでした。
今回のキーに対応するダイオードはdon-wangさんが直したダイオード（キーソケット下の一つ目）になります。

無事に解決していただきましてありがとうございます。
末永くご愛用していただけますと幸いです。
